aiounifi/models/device.py

```diff
@@ -753,6 +753,16 @@ def outlet_overrides(self) -> list[TypedDeviceOutletOverrides]:
         """Overridden outlet configuration."""
         return self.raw.get("outlet_overrides", [])
 
+    @property
+    def outlet_ac_power_budget(self) -> str | None:
+        """The amount of power available to outlets."""
+        return self.raw.get("outlet_ac_power_budget", None)
```

Comment:
```suggestion
        return self.raw.get("outlet_ac_power_budget")
```
None is default return value in get